scipy/spatial/transform/rotation.py

```diff
@@ -1461,6 +1463,43 @@ def magnitude(self):
         else:
             return angles
 
+    @classmethod
+    def create_group(cls, group, axis='Z'):
+        """Create a 3D rotation group.
+
+        Parameters
+        ----------
+        group: string
+            The name of the group. Must be one of 'I', 'O', 'T', 'Dn', 'Cn',
+            where `n` is a positive integer. The groups are:
+
+                * I: Icosahedral group
+                * O: Octahedral group
+                * T: Tetrahedral group
+                * D: Dicyclic group
+                * C: Cyclic group
+        axis: integer
+            The cyclic rotation axis. Must be one of ['X', 'Y', 'Z'] (or
+            lowercase). Default is 'Z'. Ignored for groups 'I', 'O', and 'T'.
+
+        Returns
+        -------
+        rotation : `Rotation` instance
+            Object containing the elements of the rotation group.
+
+        Notes
+        -----
+        This method generates rotation groups only. The full 3-dimensional
+        point groups [PointGroups]_ also contain reflections.
+
+        References
+        ----------
+        .. [PointGroups] `Point groups
+                          <https://en.wikipedia.org/wiki/Point_groups_in_three_dimensions>`_
+                          on Wikipedia.
+        """
```

Comment:
Can you expand the docstring with an `Examples` block that shows the usage of a 3D rotation group in a useful context? Would I potentially use it to rotate a vector? It looks like [3D rotation groups](https://en.wikipedia.org/wiki/3D_rotation_group) are composable--would it be easy to compose a rotation from several of the standard rotation groups you provide?

The object-oriented design actually makes it harder for me to think about how to do this, but anyway a simple example might be helpful at least.

Reply:
After I later looked through the tests below, I see that `g.apply(P)` might be the way to apply a single rotation group to an input vector `P`?

Reply:
Yes, that's the way to do it.

Reply:
I think the most relevant example requires the addition of the `reduce` function, which we have decided to move to a later PR.  I could create an example which takes a few vectors and rotates them by the elements of a rotation group, but it feels a bit contrived rather than being a useful example.

Can we leave the example until the next PR, or is this bad form?